packages/mantine/src/styles/Radio.module.css

```
@@ -1,10 +1,38 @@
 .labelWrapper {
     display: flex;
     align-items: flex-start;
+
+    @mixin light {
+        color: var(--coveo-color-title);
+
+        &[data-disabled] {
+            .label,
+            .description {
+                color: var(--coveo-color-text-disabled);
+            }
+        }
+    }
 }
 
 .radio {
-    @mixin where-light {
-        border-color: var(--coveo-color-input-border);
+    @mixin light {
+        &[data-error='false'] {
+            border-color: var(--coveo-color-input-border);
+        }
```

I don't see the `--coveo-color-input-border` when there's no error. I think you should use the same syntax as Mantine instead:

```suggestion
        &:not([data-error]) {
            border-color: var(--coveo-color-input-border);
        }
```

---

fixed it. thanks